workers/unity/Assets/Playground/Scripts/Worker/MobileClientWorkerConnector.cs

```diff
@@ -38,5 +65,20 @@ public override void Dispose()
 
             base.Dispose();
         }
+
+        public Option<string> GetHostIp()
+        {
+            return ipAddress;
+        }
+
+        public Option<string> GetDevAuthToken()
+        {
+            return Option<string>.Empty;
+        }
+
+        public Option<ConnectionService> GetConnectionService()
+        {
+            return Option<ConnectionService>.Empty;
```

Comment:
this should handle it based on the `shouldConnectLocally` checkbox

Reply:
What is the order of precedence? I.e. - who wins out if there are player prefs & command line args & this checkbox is set?

Reply:
Actually you kind of want three states for the checkbox as its like an override? 

- Not overridden
- Overidden and set
- Overidden and not set

Reply:
Made obsolete by the mobile launcher 🎉